csunplugged/tests/resources/management/test_makeresourcethumbnails_command.py

```diff
@@ -22,9 +22,10 @@ def test_makeresourcethumbnails_command_single_resource(self):
             "resources/grid.html",
             "GridResourceGenerator",
         )
+        # TODO: Fix these tests, they shouldn't be writing files into the build directory
```

Comment:
Make an issue for this and I can sort this out.

Reply:
Tracked by #700